src/Relation/Binary/Definitions.agda

```diff
@@ -130,6 +130,14 @@ Min R = Max (flip R)
 Minimum : Rel A ℓ → A → Set _
 Minimum = Min
 
+-- Definitions for apartness relations
+
+Comparison : Rel A ℓ → Set _
```

Comment:
Okay, more bikeshedding incoming. All the other properties names are adjectives (`Transitive`/`Symmetric`/`Total` etc.), but `Comparison` is a noun. Would it be reasonable to call this `Comparitive` instead? Or if you don't like that we could use the alternative name in the literature `Cotransitive`? The latter would also have the advantage that the shortening of it, `cotrans`, wouldn't clash with the shortening of `Trichotomous` which is `cmp`. Having two very different properties called `cmp` and `comp` probably isn't great.

Reply:
Yep agree. I've renamed the member of `IsApartnessRelation` to `cotrans`.